lib/msal-node/src/client/ManagedIdentitySources/BaseManagedIdentitySource.ts

```diff
@@ -138,6 +140,12 @@ export abstract class BaseManagedIdentitySource {
 
         const networkRequestOptions: NetworkRequestOptions = { headers };
 
+        if (managedIdentityRequest.claims) {
+            RequestValidator.validateClaims(managedIdentityRequest.claims);
```

Comment:
What does this do? Afaik claims is just a string (a json string) from MSAL's point of view. Does this need validaton?

Reply:
We validate claims in msal-node. Specifically, this method makes sure they can be parsed via `JSON.parse(claimsString)`.

Reply:
This validation adds little value and increases client latency. ESTS can reject the claims. Pls remove.